scripts/superdesk-authoring/authoring.js

```diff
@@ -384,6 +384,17 @@ import 'angular-history/history.js';
             });
         };
 
+        this.validateBeforeTansa = function (orig, diff) {
+            diff = extendItem({}, diff);
+
+            this.cleanUpdatesBeforePublishing(orig, diff);
+
+            return api.save('validate', {'act': 'publish', 'type': orig.type, 'validate': diff})
+            .then(function(result) {
```

Comment:
This `.then` statement is redundant, you can remove it. The same result will be returned.